test/integration/offchain/certificate.test.ts

```diff
@@ -6,7 +6,7 @@ import createHttpServer from '../../../src/server'
 import { put } from '../../helpers/routeHelper'
 
 import { cleanup, updateSeed } from '../../seeds/certificate'
-import { CertificateRow } from '../../../src/lib/db'
+import { CertificateRow } from '../../../src/lib/db/types'
```

Comment:
DB changes as separate PR from API controller, testing first with initiation and issuance?